server/src/internalClusterTest/java/org/elasticsearch/indices/state/CloseWhileRelocatingShardsIT.java

```diff
@@ -125,7 +125,7 @@ public void testCloseWhileRelocatingShards() throws Exception {
             final CountDownLatch release = new CountDownLatch(indices.length);
 
             // relocate one shard for every index to be closed
-            final AllocationCommands commands = new AllocationCommands();
+            final var commands = new ArrayList<AllocationCommand>();
```

Comment:
[opt/nit] since we know the size (`indices`), could this allocate a raw AllocationCommand[] array?

Reply:
Eh doing it like that involves mucking around with numeric indices, no need really.